### DISCUSSION TITLE
英特爾推出即時 Deepfake 偵測工具 FakeCatcher，準確度高達 96%

### DISCUSSION THREAD

**Discussion by neil0258** [1101-GS3073-C/Python-Community-GS3073-C#655] [General]
英特爾透過顏色細微變化即時偵測是否使用 Deepfake 技術，如果影片主體是人，變化會更明顯，因心臟跳動輸送血液時產生的血液流動會使影片畫素顏色有微妙變化，英特爾稱 FakeCatcher 可幾毫秒內就分析完畢，準確率達 96%。

我的看法：Deepfake確實荼毒且危害了許多人，即使我們再怎麼地注意自己的言行舉止及潔身自愛，還是抵擋不了有心人士動動滑鼠，敲敲鍵盤，或許就可以輕易地毀滅我們的人生；好消息是：有了FakeCatcher 這個武器，我們或許就更有機會防範不肖人士危害大眾的行為，如果再配合警檢單位，追蹤到惡意使用Deepfaker的位址，並將其逮捕。不過技術一直進步，將來 Deepfake 很有可能又能騙過 FakeCatcher。

**Comment by larry40305** [comment]
由AI所產生的犯罪行為，就由AI來取締

**Comment by HsuChen88** [comment]
我認為未來會發生AI大戰，可以請不要再給我眼睛了嗎。

**Reply by chia-kai-chang** [reply]
不要太在意啊 做自己就好

**Reply by chia-kai-chang** [reply]
我們都要開始練習做自己，我們一起加油

**Comment by bww0401** [comment]
到底是平價便宜的Deepfake技術吸引人，還是高貴奢華的FakeCatcher更勝一籌，期待未來的發展!
